delta/data/frontend/kaldi_reader_writer.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2017 Beijing Didi Infinity Technology and Development Co.,Ltd.
```

Comment:
using espnet.espnet.utils.cli_writer.py,  not add this file.

Reply:
like this https://github.com/espnet/espnet/blob/84a64f19ab/utils/apply-cmvn.py#L101